### DISCUSSION TITLE
add chinese language

### DISCUSSION THREAD

**Discussion by Halogly** [tryashtar/nbt-studio#63] [Ideas]
i want to translate this program into chinese, i can download the source code and edit it, however, i have an idea is that create a new folder that only for language in the source code, so other developers can add new languages easily, also, add a new menu to change language, have a good day! I LOVE YOU PROGRAM!

**Comment by tryashtar** [comment]
It's a great idea! Although I can't work on this anymore, I'd be happy to review any PRs that add this! :)
